algorithms/linfa-elasticnet/src/algorithm.rs

```diff
@@ -189,6 +319,92 @@ fn coordinate_descent<'a, F: Float>(
     (w, gap, n_steps)
 }
 
+fn block_coordinate_descent<'a, F: Float>(
+    x: ArrayView2<'a, F>,
+    y: ArrayView2<'a, F>,
+    tol: F,
+    max_steps: u32,
+    l1_ratio: F,
+    penalty: F,
+) -> (Array2<F>, F, u32) {
+    let n_samples = F::cast(x.shape()[0]);
+    let n_features = x.shape()[1];
```

Comment:
using `ncols` / `nrows` is a bit more expressive

Reply:
I feel like the current naming better expresses the properties of the dataset (# of rows = # of samples, # of cols = # of features). Plus this naming convention is used basically everywhere in this crate.

Reply:
nah I meant the method name, not the variable